lib/Autocomplete.js

```diff
@@ -322,6 +322,7 @@ let Autocomplete = React.createClass({
           {...this.props.inputProps}
```

Comment:
Later props take precedence: https://facebook.github.io/react/docs/jsx-spread.html#spread-attributes

This should come at the end of the props list so that the consumer can override autoComplete (and everything else for that matter).